nav2_util/include/nav2_util/lifecycle_utils.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_UTIL__LIFECYCLE_UTILS_HPP_
+#define NAV2_UTIL__LIFECYCLE_UTILS_HPP_
+
+#include <vector>
+#include <string>
+#include "nav2_util/string_utils.hpp"
+
+namespace nav2_util
+{
+
+/// Transition the given lifecycle nodes to the ACTIVATED state in order
+void bringupLifecycleNodes(const std::vector<std::string> & node_names);
```

Comment:
I agree. I wanted to make it super easy to write a test that uses lifecycle nodes, because I expect we will want to do that a lot to improve test stability.

I also didn't want to invest too much time into managing lifecycle states until you'd had a chance to look at this. I figured you probably had a better way to do this in the work you'd been doing.

Reply:
As I was went through this to separate test code from production code, I found I didn't have any test code to put in the library, so I'm going to defer creating that library.

This `bringupLifecycleNodes` function ended up being used by the command line utility I created, which I consider production code as opposed to just test code.